kubernetes/assets/monitors/monitor_node_unavailable.json

```diff
@@ -1,8 +1,8 @@
 {
-	"name": "[kubernetes] Monitor Unavailable Kubernetes Nodes",
+	"name": "[kubernetes] Monitor Unschedulable Kubernetes Nodes",
 	"type": "query alert",
 	"query": "max(last_15m):sum:kubernetes_state.node.status{status:schedulable} by {kubernetes_cluster} * 100 / sum:kubernetes_state.node.status{*} by {kubernetes_cluster} < 80",
-	"message": "Nodes are unavailable on ({{kubernetes_cluster.name}} cluster)",
+	"message": "More than 20% of nodes are unschedulable on ({{kubernetes_cluster.name}} cluster). \n Keep in mind that this might be expected based on your infrastructure.",
```

Comment:
could we link to online resources that would help whoever receive that alert?

Reply:
The only 'official' resources I could find was https://kubernetes.io/docs/concepts/architecture/nodes/ which mentions the following `Marking a node as unschedulable prevents the scheduler from placing new pods onto that Node, but does not affect existing Pods on the Node. This is useful as a preparatory step before a node reboot or other maintenance.`
@hmakkapati are there any docs that you're aware of for this?

Reply:
A node is unschedulable when it is cordoned. As far as I'm aware, there aren't many things in kube that automatically corodn a node. Kube might add 'NoSchedule' taints on certain conditions when a node misbehaves but that doesn't make the node  "unschedulable". So, unschedulable nodes in most cases are a result of some manual/automated actions, which are very specific to each operator and the remedy would revolve around the reason for cordoning.

Reply:
Gotcha - leaving the text as is since there's no definite next steps we can provide.